front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/Widgets/FollowedItems/index.tsx

```diff
@@ -18,12 +20,18 @@ interface Props {
 
 const FollowedItems = ({ titleMultiloc }: Props) => {
   const localize = useLocalize();
+  const { data: authUser } = useAuthUser();
```

Comment:
We only show the skeleton here for signed in users, because for signed out users there will never be any followed items.